src/utils/appLinking.js

```diff
@@ -0,0 +1,117 @@
+import { Linking } from "react-native";
+import { Alert } from "../components/general/Alert";
+
+/**
+ * Object that holds the slack team id and several channel ids.
+ * The information in here is the most current as of May 28th, 2020.
+ * to find the team id or channel id login in slack on a computer.
+ * The url will look like this https://app.slack.com/client/{team_id}/{channel_id}
+ */
+
+export const slackInfo = {
+
+	team: "TC61JSPUZ",
+	channel: {
+		general: "CPRDXHHD4",
+		announcements: "CC5S86GHE",
+		fundraisingCommitte: "CCUUWRU1Y",
+		motorshpe: "CP5HC76TD",
+		projects: "CNATZRD9C",
+		shpejr: "GCWFE5630"
+	}
+};
+
+/**
+ * @description This function handles the action launching an app, like a mail app, from our app.
+ *
+ *  ** It requires to pass an url and optional warning message.**
+ *  Look under URL formats to see examples of correctly formatted urls for different apps.
+ *  When dealing with slack, import slackInfo to gain access of the team id and channel id.
+ *
+ *      @param {String}  url           The url of the app to be open.
+ *      @param {Stirng=} message       Optional value for the alert. It use to populate the alert dialog box.
+ *
+ * @example
+ *  URL formats
+ *      slack:
+ *              To open the chapter worspace  ->  https://www.slack.com/open?team={slackInfo.team}
+ *              to send the user directly to a channel,
+ *              let's say general             ->  https://www.slack.com/channel?team={slackInfo.team}&id={slackInfo.channel.general}
+ *      linked:
+ *              To open a profile page        ->  https://www.linkedin.com/in/{profile}
+ *              To open a company page        ->  https://www.linkedin.com/company/{companyName}
+ *      instagram:
+ *              To open a profile page        ->  https://www.instagram.com/{profile}
+ *      facebook:
+ *              To open a group page          ->  https://www.facebook.com/groups/{groupName}
+ *      email:
+ *              To send an email              ->  mailto:{email}
+ *      phone:
+ *              To make a call, let's say to 407-555-5555
+ *                                            ->  tel:{+14075555555}
+ *      website: websites needs to end with "/"
+ *                                            ->  https://www.shpeucf.com/
+ *
+ *  -- How to use the function --
+ *  Calling the function passing both params, the url and the alert message. -> appLinkingHandler(url, message)
+ *
+ *  Calling the function passing one param, the url. -> appLinkingHandler(url)
+ */
```

Comment:
Overall this documentation looks pretty good, Idel might come and point out small things that I don't really notice. Also when you change the name of appLinkingHandler, Make sure you change it  here too.

Reply:
I agree the documentation looks pretty good. The only thing I'd say here is that we can maybe simplify the how to use section to one line, briefly identifying the message param as optional.